templates/common/.azdo/pipelines/bicep/azure-dev.yml

```diff
@@ -10,36 +10,33 @@ trigger:
 pool:
   vmImage: ubuntu-latest
 
-# Use azd provided container image that has azd, infra, multi-language build tools pre-installed.
-container: mcr.microsoft.com/azure-dev-cli-apps:latest
-
 steps:
+  - task: setup-azd@0
```

Comment:
Can you add a comment about this task needs to be manually installed in customer's devops instance?
Maybe a link to the extension or to azd-official docs

Reply:
@vhvb1989 I have updated it based on your comments. Please review.